src/frontend/src/flows/addDevice/welcomeView/deviceRegistrationModeDisabled.ts

```diff
@@ -22,8 +21,8 @@ const deviceRegistrationDisabledInfoTemplate = ({
     </hgroup>
     <ol class="c-list c-list--numbered l-stack">
       <li>
-        Connect to ${LEGACY_II_URL_NO_PROTOCOL} on a recognized device using
-        Internet Identity ${userNumber}
+        Connect to ${window.origin} on a recognized device using Internet
```

Comment:
note: this now includes the protocol. I think originally we assumed this was purely informative and not really relevant in the dev build. Not saying we shouldn't make the change.

It's also slightly unfortunate that the screen (e.g. in showcase) is now dynamic and depends on where the page is served. Might be nice to pass in the origin as a parameter instead.

Reply:
I think including the protocol is ok. It's also nice, that it points to the correct location for staging, etc. deployments.

I added the parameter.